src/grammar_io.jl

```diff
@@ -1,9 +1,9 @@
 """
-    store_cfg(filepath::AbstractString, grammar::ContextFreeGrammar)
+    store_cfg(filepath::AbstractString, grammar::ContextSensitiveGrammar)
```

Comment:
There are now both the cfg methods and csg method (in name), both using ContextSensitiveGrammar, I think you can remove half of them? Or name them all csg if there is different functionality?

Reply:
I merged the CFG functions into CSG ones. I had to change the type signature of store_csg slightly as to make the optional argument last. The tutorials on grammar IO should be changed to not invoke the removed functions and correctly call store_csg.